app/Http/Controllers/Auth/RegisteredUserController.php

```diff
@@ -43,13 +56,26 @@ public function store(Request $request): RedirectResponse
             'g-recaptcha-response' => app()->environment('production') ? ['required', new Recaptcha($request->ip())] : [],
         ]);
 
+        $request->session()->forget(['github_email', 'github_username']);
```

Comment:
If the validation was successful, we don't need these two fields in the session any longer